lib/vsc/filesystem/posix.py

```diff
@@ -558,3 +559,69 @@ def create_stat_directory(self, path, permissions, uid, gid, override_permission
             self.log.debug("Path %s already exists with correct ownership", path)
 
         return created
+
+    def replace_acl(self, path, permissions):
+        """
+        Overwrite ACL permissions on given path
+        """
+        path = self._sanity_check(path)
+        fs = self._what_filesystem(path)
+        if fs is None:
+            self.log.warning(f"ACL replacement requested on  filesystem with unknown support: {path}")
```

Comment:
We're going to purge all this at some point and use logging.xyz. Not sure if you want to change this already then.

Reply:
It's confusing to mix `self.log` and `logging` in the same code. So I won't make that change just here. When the switch is made to logging we can do that everywhere at once.